src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/tools/PStack.java

```diff
@@ -82,6 +82,7 @@ public void run(PrintStream out, Debugger dbg) {
            return;
         }
          final boolean cdbgCanDemangle = cdbg.canDemangle();
+         String fillerForAddress = " ".repeat(2 + (2 << VM.getVM().getLogAddressSize())) + "\t";
```

Comment:
Why not use `2 * VM.getVM().getAddressSize()` rather than `(2 << VM.getVM().getLogAddressSize()`?

Reply:
Thank you. I missed that method. I've changed the code.